package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@patternfly/patternfly": "^2.23.0",
```

Comment:
Is it better to fix the version of this?

```suggestion
"@patternfly/patternfly": "2.23.0"
```

Reply:
yep all these should be fixed.  Will update these thanks